Contrib/CLI/wcli.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+function config_extract() {
+    echo $(cat ~/.walletwasabi/client/Config.json | jq -r "$1")
+}
+
+ENABLED="$(config_extract '.JsonRpcServerEnabled')"
+CREDENTIALS=$(config_extract '.JsonRpcUser + ":" + .JsonRpcPassword')
+ENDPOINT=$(config_extract '.JsonRpcServerPrefixes[0]')
+BASIC_AUTH=$([ "$CREDENTIALS" == ":" ] && echo "" || echo "--user ${CREDENTIALS}")
+
+WALLETNAME=""
+
+if [  $# -ge 1 ]; then
+    ARG="${1%=*}"
+    if [[ "$ARG" == "-wallet" ]]; then
+        WALLETNAME="${1#*=}/"
+        shift
+    fi
+fi
+
+METHOD=$1
+shift
+
+if [ $# -ge 1 ]; then
+    if [[ "$1" ]]; then
+        PARAMS="\"$1\""
+        shift
+    else
+        PARAMS="\"\""
+        shift
+    fi
+
+    while (( "$#" )); do
+        if [[ "$1" ]]; then
+            PARAMS="$PARAMS, $1"
+        else
+            PARAMS='$PARAMS, ""'
+        fi
+        shift
+    done
+fi
+
+REQUEST="{\"jsonrpc\":\"2.0\", \"id\":\"curltext\", \"method\":\"$METHOD\", \"params\":[$PARAMS]}"
+RESULT=$(curl -s "$BASIC_AUTH" --data-binary "$REQUEST" -H -- "content-type: text/plain;" "$ENDPOINT$WALLETNAME")
+RESULT_ERROR=$(echo "$RESULT" | jq -r .error)
+
+rawprint=(help)
+
+if [[ "$RESULT_ERROR" == "null" ]]; then
+    if [[ " ${rawprint[@]} " =~ " ${METHOD} " ]]; then
```

Comment:
There is ShellCheck errors with this.
```
    if [[ " ${rawprint[@]} " =~ " ${METHOD} " ]]; then
          ^----------------^ SC2199 (error): Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).
                                ^-----------^ SC2076 (warning): Remove quotes from right-hand side of =~ to match as a regex rather than literally.
```

Reply:
That's for help method only which doesn't exist yet. I changed the line but not sure it is correct.